src/page/resolver/wrapperResolver.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default (type) => {
+  switch(type) {
+    case 'add':
+      return 'form';
+    case 'edit':
+    case 'list':
+    case 'show':
+      return 'div';
```

Comment:
don't see test for this case.

Reply:
it's temporary due to other tests. should be throwing exception but then other tests are not passing due to dependency to this resolver